src/timesync/timesyncd-bus.c

```diff
@@ -238,6 +240,78 @@ static const sd_bus_vtable manager_vtable[] = {
         SD_BUS_VTABLE_END
 };
 
+static int reload_dbus_and_request(sd_bus *bus) {
+        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+        int r;
+
+        r = sd_bus_call_method(
+                        bus,
+                        "org.freedesktop.DBus",
+                        "/org/freedesktop/DBus",
+                        "org.freedesktop.DBus",
+                        "ReloadConfig",
+                        &error, NULL, NULL);
+        if (r < 0)
+                return log_error_errno(r, "Failed to reload DBus configuration: %s", bus_error_message(&error, r));
```

Comment:
hmm, this feels a bit hacky to me.. Should at least contain a comment i figure explaining why we have this...

Also, is ReloadConfig() itself not protected? i.e. can we call this with normal privileges?

Reply:
hmm, one more tweak please: let's enqueue the method call here asynchronously, instead of waiting for it. We spend so much time in making the start-up phase of our various daemons async, so let's not reintroduce synchronization points needlessly.